diff --git a/src/Client.php b/src/Client.php
index 315a022c..bbba32c9 100644
--- a/src/Client.php
+++ b/src/Client.php
@@ -442,7 +442,7 @@ class Client implements ClientInterface
         if (isset($options['query'])) {
             $value = $options['query'];
             if (is_array($value)) {
-                $value = http_build_query($value, null, '&', PHP_QUERY_RFC3986);
+                $value = http_build_query($value, '', '&', PHP_QUERY_RFC3986);
             }
             if (!is_string($value)) {
                 throw new \InvalidArgumentException('query must be a string or array');
